example/javascriptlib/basic/1-simple/build.mill

```diff
@@ -1,18 +1,159 @@
 package build
-import mill._, javascriptlib._
+import mill._
+
+trait TypeScriptModule extends Module {
```

Comment:
You should use `TypeScriptModule` from `javascriptlib` instead of copying it all here

Reply:
using `import mill._, javascriptlib._`

exits with this error:

```
[build.mill-58] [error] /.../build.mill:3:16: not found: value javascriptlib
[build.mill-58] [error] import mill._, javascriptlib._

```

Reply:
note, import scalalib._ and others works fine

Reply:
I think documentation https://mill-build.org/mill/index.html
should include dist.run usage as it was my primary source for information and I am sure it would be for others